README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Flutter can be used to build applications for Android and IOS
+- The Dart language used in Flutter is quite similar to Java
+- We can update the application in real time
+- If we close the application when we open it again we can continue from where we were
```

Comment:
This benefit is not _exclusive_ to Flutter development and it's _unclear_ what this means. 
Are you saying that you can pickup your work in the IDE or When you re-open the test app on the _device_?

Reply:
I was talking about getting us back to where we were at the IDE.

But it probably wasn't the best advantage of using Flutter.
There is also the "Hot reload" advantage that allows the developer to see changes made only to what was modified without having to run the application from the beginning again.
Flutter reloads only the screen that was changed and faster.

Reply:
All modern IDEs have this feature so that people don't lose their work when the IDE or OS crashes.
Try opening a Phoenix project in Atom. Open a bunch of files in several tabs. 
Quit the Atom and re-open it. You'll see that the active tab will still be active when you re-open the app.

Hot reloading _is_ a major benefit of Dart/Flutter. 👍